examples/intel/file/file.py

```diff
@@ -14,7 +14,7 @@ def main():
     print(f"Checking hash...")
 
     try:
-        response = intel.lookup(
+        response = intel.hashReputation(
```

Comment:
hashReputation sounds a bit funny, can we call this file_hash_reputation (lets try to keep the language naming convention by using snake-case

Reply:
You are right, this is not the right convention for python. I probably mixup with Node or Java. But these changes were already released on SDK last version.